app/mailers/instructor_notification_mailer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class InstructorNotificationMailer < ApplicationMailer
+  def self.send_email(alert)
+    return unless Features.email?
+    email(alert).deliver_now
+  end
+
+  def email(alert)
+    @alert = alert
+    set_email_parameters
+    params = { to: @instructors.pluck(:email),
```

Comment:
This should probably BCC to the sender, to make it easy to know that it went through.